spaceship/lib/spaceship/connect_api/models/user.rb

```diff
@@ -57,6 +57,30 @@ def self.find(client: nil, email: nil, includes: ESSENTIAL_INCLUDES)
         return all(client: client, filter: { email: email }, includes: includes)
       end
 
+      # @param client [ConnectAPI] ConnectAPI client.
+      # @param all_apps_visible [Boolean] If all apps must be visible to a user. true - if a user must see all apps, you must not provide visible_app_ids, `false` - a user must see only a limited list of apps, and you must provide visible_app_ids. nil if no change is needed.
+      # @param provisioning_allowed [Bool] If a user with a Developer or App Manager role must have access to Certificates, Identifiers & Profiles. true - if a user must be able to create new certificates and provisioning profiles, `false` - otherwise. nil if no change is needed.
+      # @param roles [Array] Array of strings describing user roles. Possible values: https://developer.apple.com/documentation/appstoreconnectapi/userrole.nil if no change is needed.
```

Comment:
There is a module `Spaceship::ConnectAPI::UserInvitation::UserRole` with constants (which is never used, btw). I think we can mention it in the first place alongside with link to the official Apple docs. What about this correction

```suggestion
      # @param roles [Array] Array of strings describing user roles. Possible values are described in Spaceship::ConnectAPI::UserInvitation::UserRole, or you can refer to https://developer.apple.com/documentation/appstoreconnectapi/userrole. Pass nil if no change is needed.
```

Reply:
The only mention of roles as hashes I [found in Tunes](https://github.com/fastlane/fastlane/blob/3b937c0ef9370b5b18dbdae4c83089af31e540d0/spaceship/lib/spaceship/tunes/member.rb#L22-L45).